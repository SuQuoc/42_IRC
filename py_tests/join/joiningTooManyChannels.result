:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:client0!user0@10.14.3.9 JOIN #chan0 * :user0
:client0!user0@10.14.3.9 JOIN #chan1 * :user0
:client0!user0@10.14.3.9 JOIN #chan2 * :user0
:client0!user0@10.14.3.9 JOIN #chan3 * :user0
:client0!user0@10.14.3.9 JOIN #chan4 * :user0
:client0!user0@10.14.3.9 JOIN #chan5 * :user0
:client0!user0@10.14.3.9 JOIN #chan6 * :user0
:client0!user0@10.14.3.9 JOIN #chan7 * :user0
:client0!user0@10.14.3.9 JOIN #chan8 * :user0
:client0!user0@10.14.3.9 JOIN #chan9 * :user0
:AfterLife 405 client0 #chan10 :You have joined too many channels
:client0!user0@10.14.3.9 QUIT :Leaving
:client0!user0@10.14.3.9 QUIT :Leaving
:client0!user0@10.14.3.9 QUIT :Leaving
:client0!user0@10.14.3.9 QUIT :Leaving
:client0!user0@10.14.3.9 QUIT :Leaving
:client0!user0@10.14.3.9 QUIT :Leaving
:client0!user0@10.14.3.9 QUIT :Leaving
:client0!user0@10.14.3.9 QUIT :Leaving
:client0!user0@10.14.3.9 QUIT :Leaving
:client0!user0@10.14.3.9 QUIT :Leaving
