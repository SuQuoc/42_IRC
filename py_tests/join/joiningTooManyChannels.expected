:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 005 client0 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:client0!user0@10.12.3.8 JOIN #chan0 * :user0
:client0!user0@10.12.3.8 JOIN #chan1 * :user0
:client0!user0@10.12.3.8 JOIN #chan2 * :user0
:client0!user0@10.12.3.8 JOIN #chan3 * :user0
:client0!user0@10.12.3.8 JOIN #chan4 * :user0
:client0!user0@10.12.3.8 JOIN #chan5 * :user0
:client0!user0@10.12.3.8 JOIN #chan6 * :user0
:client0!user0@10.12.3.8 JOIN #chan7 * :user0
:client0!user0@10.12.3.8 JOIN #chan8 * :user0
:client0!user0@10.12.3.8 JOIN #chan9 * :user0
:AfterLife 405 client0 #chan10 :You have joined too many channels
